chain/messagepool/messagepool.go

```diff
@@ -52,6 +53,8 @@ var minimumBaseFee = types.NewInt(uint64(build.MinimumBaseFee))
 
 var MaxActorPendingMessages = 1000
 
+var MaxNonceGap = uint64(runtime.NumCPU())
```

Comment:
this is the number of validator goroutines in pubsub, we all this slack to account for concurrent message validation of otherwise consecutive messages.

Reply:
I think we might be able to tighten this if we consider the number of messages that can fit in a packet; so a cap of 16 or so might be in order

Reply:
Tightened it to `min(NumCPU, 16)`.

Reply:
Thinking more about it, we can set it to a very low hard cap (say 4) and induce some (small) delay when republishing messages to make that they get through serially with high probability.

Reply:
(I really don't like having a spam attack surface at all, even if it's just a few messages; 4 is too small to make a dent)